### PR TITLE
Update use of deprecated method.

### PR DESCRIPTION
**Pull request by ADmad**
Refs #751.

### CODE REVIEW

File: tests/comparisons/Model/testBakeTableValidation.php [hunk 1]
```diff
@@ -80,7 +80,10 @@ public function validationDefault(Validator $validator): Validator
             ->allowEmptyString('email');
 
         $validator
-            ->uploadedFile('image', ['optional' => true, 'types' => ['image/jpeg']])
+            ->uploadedFile('image', [
```
Comment (ADmad) [inline tests/comparisons/Model/testBakeTableValidation.php:83]: Currently `VarExporter` doesn't have an option to force generation of a single line representation of associative array. I'll see if I can they would be interested in adding such an option.

Comment (othercorey) [reply tests/comparisons/Model/testBakeTableValidation.php:83]: They might not want to support things like "maximum line length" which people might ask for with that.

Comment (ADmad) [reply tests/comparisons/Model/testBakeTableValidation.php:83]: Yeah the that might not be feasible, so inline associative arrays should be done only when it's know beforehand that the result won't exceed "max line length".

Comment (markstory) [reply tests/comparisons/Model/testBakeTableValidation.php:83]: Multi line arrays are fine though. I don't think we need to stick to single line output.

Comment (ADmad) [reply tests/comparisons/Model/testBakeTableValidation.php:83]: Yup, my initial comment was more of an informative one just to clarify why the output changed :slightly_smiling_face:.

**Pull request merged**